package.json

```
@@ -26,7 +26,7 @@
     "styled-components": "^5.0.0 || ^6.0.0"
   },
   "dependencies": {
-    "hpe-design-tokens": "~1.4.0"
+    "hpe-design-tokens": "https://github.com/grommet/hpe-design-system/tarball/design-tokens-v2-alpha-stable"
```

NOTE @jcfilben @britt6612 Before merging, we should update this to:

```suggestion
    "hpe-design-tokens": "~2.0.0"
```